hail/src/main/scala/is/hail/expr/ir/InferPType.scala

```diff
@@ -372,14 +372,18 @@ object InferPType {
           theIR._pType2
         }))
       case In(_, pType: PType) => pType
-
+      case ArrayFor(a, valueName, body) => {
```

Comment:
remove the braces here and in the ResultOp case, it's against the style convention in our codebase